.github/ISSUE_TEMPLATE/3_technical_task.md

```diff
@@ -0,0 +1,21 @@
+---
+name: 🔨 Technical task
+about: Record an action that needs to be taken
```

Comment:
a generic description; state what qualifies a ticket as a technical task rather than something else e.g. not user-facing or visible in a UI or ...